### ISSUE TITLE
About minimum libc version supported

### ISSUE THREAD

**Issue by genaris** [hyperledger/anoncreds-rs#105]
Today, when integrating anoncreds-rs in AFJ, CI/CD failed with the following error:

```
Dynamic Linking Error: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /home/runner/work/aries-framework-javascript/aries-framework-javascript/node_modules/@hyperledger/anoncreds-nodejs/native/libanoncreds.so)
```

CI was set up to use a container based on Ubuntu 20.04, whose libc version is older than 2.33 (as this version was released in 2021). Our particular issue was solved by upgrading to Ubuntu 22.04, but I would like to know if this is on purpose or it is possible to use an older libc version.

I noticed that in both aries-askar and indy-vdr, cross is used when building for Linux targets. There is a comment stating: _using cross here to build against an older glibc for compatibility_. 

Should we use cross here as well to support older platforms?

**Comment by blu3beri** [comment]
I don't think that it is done on purpose here and if using cross will allow for compatibility with ubuntu 20.04 and other slightly older platforms we should definitely add it in this repository as well. 

@andrewwhitehead was that the reason `cross` was used for aries-askar and indy-vdr?

**Comment by swcurran** [comment]
@blu3beri — is this fixed now?

**Comment by blu3beri** [comment]
Have not taken a look yet so I doubt it. For sure something that needs resolving.

**Comment by TimoGlastra** [comment]
Not fixed yet!
